ffdc.sh

```diff
@@ -24,7 +28,7 @@ function close_ffdc {
 }
 
 function fail_unless_bastion {
```

Comment:
This function must be renamed to something like "fail_retrieving_project" since we are no longer using a bastion node anymore. The current name could be misleading. Also, all the references to this function need to be changed with the new name.

Reply:
Adressed, renamed to fail_unless_oc_cli